docs/server/installation.rst

```diff
@@ -280,6 +280,13 @@ setting appropriately in your custom settings file, ensuring that you are using
 the correct :setting:`ENGINE <DATABASE-ENGINE>` setting for your chosen
 database backend.
 
+Pootle requires `django-transaction-hooks <https://pypi.python.org/pypi/django-transaction-hooks/>`_
+to connect to the database. The following database backends are supported:
+
+  - sqlite: transaction_hooks.backends.sqlite3
+  - mysql: transaction_hooks.backends.mysql
+  - postgres: transaction_hooks.backends.postgresql_psycopg2
```

Comment:
It is still necessary to include this warning on the release notes. Users upgrading must not miss this important change.

Since it is an important change I also would like to see some checks on https://github.com/translate/pootle/blob/master/pootle/checks.py to ensure the old values (regular Django database backends) are no longer used.

I suggest this to be converted into a warning to catch the reader's eye, move it to the beginning of the section, and highlight `transaction_hooks.backends.sqlite3` and relatives.